tests/util/acquisition/sampler.py

```diff
@@ -1,7 +1,7 @@
 import tensorflow as tf
 
 from tests.util.models.gpflow.models import QuadraticMeanAndRBFKernel
-from trieste.acquisition.sampler import BatchReparametrizationSampler
+from trieste.models.gpflow import BatchReparametrizationSampler
```

Comment:
Can you write somewhere (either in the PR notes or in the commit log) what breaking import changes there are so we can include them in the next release notes?